zebra-chain/src/orchard/sinsemilla.rs

```diff
@@ -191,8 +191,8 @@ pub fn sinsemilla_short_commit(
 }
 
 // TODO: test the above correctness and compatibility with the zcash-hackworks test vectors
-// https://github.com/ZcashFoundation/zebra/issues/2079
-// https://github.com/zcash-hackworks/zcash-test-vectors/pulls
+// <https://github.com/ZcashFoundation/zebra/issues/2079>
```

Comment:
Can you confirm that the link in docs that start with `//` like this one instead of `///` or `//!` also produce warnings ?

Reply:
Comments starting with `//` don't produce warnings, but I think it's useful to have them in the correct format in case we add a `/` when we want to propagate them to the docs.

Reply:
Yea, well, i guess is the same, i was just thinking that from now on we should block any PR that uses a link without `<` and `>` in any comment and not just in rustdoc comments. Should we write this somewhere as a contributor guideline ?

Reply:
I am more in the side of don't make the change or force users to do it certain way if it is not needed but i am ok either way. @dconnolly do you have an opinion.

Reply:
I was doing a quick check around and just found that the `cargo` rust project does not use the `<link>` format in normal comments: 

- https://github.com/rust-lang/cargo/blob/master/src/cargo/core/features.rs#L134
- https://github.com/rust-lang/cargo/blob/master/src/cargo/core/profiles.rs#L274

But it definitly do so in rustdocs:

- https://github.com/rust-lang/cargo/blob/master/src/cargo/core/features.rs#L22

Reply:
@oxarbitrage do you think we should remove the format from normal comments, or can we mark this as solved? I don't mind either way. It seems easier to merge this as is, and when we add the check to CI the normal comments won't be enforced anyway, so it won't block if people use the format in those.

Reply:
I think we should not use this format in normal comments so that change on my opinion needs to be reversed. 

I am also unsure on how is the workflow between this PR and https://github.com/ZcashFoundation/zebra/pull/4514

Reply:
> I think we should not use this format in normal comments so that change on my opinion needs to be reversed.

OK, I will reverse it.

> I am also unsure on how is the workflow between this PR and https://github.com/ZcashFoundation/zebra/pull/4514

I thought I'd rebase that PR once this is merged. That PR will be much easier to review then.

Reply:
While reversing the normal comments, I noticed we use Markdown syntax for other parts of the comments, even though the comments are not rendered in the docs. A part of the Markdown syntax is to use `<` and `>` for links: https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses. Don't we create an inconsistency if we don't use the angle brackets?

Reply:
I dont think we do because normal comments don't have to be markdown at all.

Reply:
Done in [f567869](https://github.com/ZcashFoundation/zebra/pull/4494/commits/f567869478277cb213d8462a327d2ecdc3ee7d39).